TF_Tooling_Web/src/Cursors/DotCursor.ts

```diff
@@ -62,7 +62,7 @@ export class DotCursor extends TouchlessCursor {
     // Function: UpdateCursor
     // Used to update the cursor when recieving a "MOVE" <ClientInputAction>. Updates the
     // cursor's position, as well as the size of the ring based on the current ProgressToClick.
-    UpdateCursor(_inputAction: TouchFreeInputAction): void {
+    protected UpdateCursor(_inputAction: TouchFreeInputAction): void {
```

Comment:
💬 Is this changed to protected for the cursor test?

Reply:
Changed to protected as I don't think these methods should ever be called outside of the cursor classes